utam-compiler/src/main/java/utam/compiler/grammar/UtamElement.java

```diff
@@ -293,8 +293,7 @@ private Basic() {
       LocatorCodeGeneration locatorHelper = selector.getCodeGenerationHelper(context);
       MethodParametersTracker allParameters = new MethodParametersTracker(String.format("element '%s' getter", name));
       MethodParametersTracker parametersWithoutFilter = new MethodParametersTracker(String.format("element '%s' getter", name));
-      if(scopeElement != null) {
-        allParameters.setMethodParameters(scopeElement.getParameters());
```

Comment:
This line is redundant. The constructor for `ElementContext` will add the scope element parameters into the parameter list for this element.

Reply:
bug happened because of my refactoring to track unique parameters. i reverted it to be more similar to what it was prior refactoring but keeping track of unique names. thx for fixing.